docs/metrics/extending-the-sdk/README.md

```diff
@@ -72,12 +71,88 @@ to the `MeterProvider` as shown in the example [here](./Program.cs).
 
 Not supported.
 
-## ExemplarFilter
+## ExemplarReservoir
 
-Not supported.
+> [!NOTE]
+> `ExemplarReservoir` is an experimental API only available in pre-release
+  builds. For details see:
+  [OTEL1004](../../diagnostics/experimental-apis/OTEL1004.md).
 
-## ExemplarReservoir
+Custom [ExemplarReservoir](../customizing-the-sdk/README.md#exemplarreservoir)s
+can be implemented to control how `Exemplar`s are recorded for a metric:
```

Comment:
perhaps a note here asking users to open an issue with the need of authorizing custom reservoirs, so we'll be able to get a feel of what are the things users are after....

Reply:
Added